sdk/storage/Azure.Storage.Files/tests/UnitTests.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Azure.Storage.Common.Tests.Shared;
+using Azure.Storage.Files.Models;
+using NUnit.Framework;
+
+namespace Azure.Storage.Files.Tests
+{
+    public class UnitTests
+    {
+        [Test]
+        public void StorageFileDownloadInfo_Dispose()
+        {
+            MockStream stream = new MockStream();
+            StorageFileDownloadInfo storageFileDownloadInfo =
```

Comment:
Do we use `var` in the tests?

Reply:
I think we still get the editorconfig warning to use explicit type instead of 'var' even in tests.